link/kprobe_multi.go

```diff
@@ -167,9 +167,11 @@ var haveBPFLinkKprobeMulti = internal.FeatureTest("bpf_link_kprobe_multi", "5.18
 	if errors.Is(err, unix.EINVAL) {
 		return internal.ErrNotSupported
 	}
-	if err != nil {
-		return err
+	// If CONFIG_FPROBE isn't set.
+	if errors.Is(err, unix.ENOTSUP) {
+		return internal.ErrNotSupported
 	}
+
```

Comment:
shouldn't the `if err != nil` check be left here?

Reply:
:facepalm: Yes, my bad. Good catch. Fixed.